Swipeable.js

```diff
@@ -218,28 +222,28 @@ export default class Swipeable extends Component<PropType, StateType> {
     }).start(({ finished }) => {
       if (finished) {
         if (toValue > 0 && this.props.onSwipeableLeftOpen) {
-          this.props.onSwipeableLeftOpen();
+          this.props.onSwipeableLeftOpen(fromValue, velocityX);
```

Comment:
Can we make it an object? 
e.g.
```suggestion
          this.props.onSwipeableLeftOpen({ fromValue, velocityX });
```

I'm not supportive of relying on the order of arguments in such cases. E.g. If I'll need `velocityX` and not `fromValue` then I'll need to write: `onSwipeableLeftOpen = (_, velocityX) => {}`